playbooks/spark-unit-test-arm64/run.yaml

```diff
@@ -0,0 +1,39 @@
+- hosts: all
+  become: yes
+  roles:
+    - install-openjdk
+    - install-maven
+
+- hosts: all
+    tasks:
+      - name: 'Step1: Try to build Spark directly'
+        shell: |
+          set -xo pipefail
+          # build and test spark
+          ./build/mvn -DskipTests clean package 2>&1 | tee "{{ ansible_user_dir }}/workspace/logs/spark_build_original.log"
+          # mvn test will failed due to the leveldbjni has no aarch64 supporting
+          ./build/mvn test 2>&1 | tee "{{ ansible_user_dir }}/workspace/logs/spark_test_original.log"
+        args:
+          executable: /bin/bash
+          chdir: '{{ zuul.project.src_dir }}'
+        ignore_errors: yes
+        environment: '{{ global_env }}'
+
+      - name: 'Step2: Build and run unit tests for apache spark after fixing errors'
+        shell:
+          cmd: |
+            set -ex
```

Comment:
we also need pipefail like L11, right?

Reply:
I don't think we have to set -o, why it needed?